src/microbe_stage/Patch.cs

```diff
@@ -208,6 +208,50 @@ public long GetSpeciesPopulation(Species species)
         return population;
     }
 
+    public float GetCompoundAmount(string compoundName)
+    {
+        var compound = SimulationParameters.Instance.GetCompound(compoundName);
+
+        switch (compoundName)
+        {
+            case "sunlight":
+                return Biome.Compounds[compound].Dissolved * 100;
+            case "oxygen":
+                return Biome.Compounds[compound].Dissolved * 100;
+            case "carbondioxide":
+                return Biome.Compounds[compound].Dissolved * 100;
+            case "nitrogen":
+                return Biome.Compounds[compound].Dissolved * 100;
```

Comment:
Very small tweak I'd make is maybe turning the first three case statements to fallthrough.

Reply:
Sure, that sounds doable.

Reply:
Done: https://github.com/Revolutionary-Games/Thrive/pull/3120/commits/d5739e88ba9ad9ded45313d464a0105a139b7281
though, this was code I just moved so someone else was the original author of this code.